### ISSUE TITLE
Use score to calculate the accuracy

### ISSUE THREAD

**Issue by jonathangomz** [jonathangomz/PythonClassifier#1]
We going to add the function to calculate the accuracy using the method score() from sklearn.
